src/styles/card.scss

```diff
@@ -0,0 +1,83 @@
+.card {
+  box-sizing: border-box;
+  border: 1px solid #f3f3f3;
+  font-family: $bodyFont;
+  padding: 32px 16px 16px;
+  height: 406px;
+  transition-duration: 300ms;
+
+  &:hover {
+    transform: scale(120%);
+    color: #34568b;
+  }
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    display: block;
+    margin: 0 auto 40px;
+  }
+  &__title {
+    color: #060b35;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    margin-bottom: 4px;
+  }
+  &__code {
+    color: #616070;
+    text-align: left;
+    margin-bottom: 16px;
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+  }
+  &__button {
+    color: #fff;
+    background-color: #00acdc;
+    border-radius: 5px;
+    width: 100%;
+    height: 40px;
+    font-size: 14px;
+    font-weight: 700;
+    line-height: 16px;
+    border: 1px solid #00acdc;
+    font-family: $bodyFont;
```

Comment:
The font-family is already defined in the `.card` class. There's no need to redeclare it here unless you intend to use a different font for the button.